types/set/README.md

```diff
@@ -0,0 +1,137 @@
+# Set
+
+_Module_
+
+Implements a very simple `SetInt` struct.
+
+<br>
+This is NOT a Mojo builtin, so it needs to be imported
+<br>
+
+### **Parameters:**
+
+-   None
+
+### **Fields:**
+
+-   None
+
+### **Functions:**
+
+#### `__init__`
+
+> `fn __init__(inout self) -> None:`
+
+Constructs a new `SetInt` with a default capacity (capacity_default = 11).
```

Comment:
May I ask why the default is 11 rather than a nice round number like 10? Not annoyed or anything, just curious :D.

Reply:
The idea was just to use a prime number to start to reduce a little bit the collisions that can happen (i don't think it works in reality i just used as an idea, because this is a very very simple set)